components/SearchSuggestion/lib/util.ts

```diff
@@ -26,3 +29,27 @@ export const buildPageUrlFromSuggestion = (suggestion: Suggestion) => {
       return `/question/${questionSuggestion.id}/${questionSuggestion.slug}`;
   }
 };
+
```

Comment:
Note the function below returns "Safe" HTML. HTML and not string because we are using javascript replace function to replace portions with highlighted equivalent (this requires HTML)